lib/py/test/test_sslsocket.py

```diff
@@ -213,11 +213,9 @@ def test_unix_domain_socket(self):
             return
         fd, path = tempfile.mkstemp()
         os.close(fd)
-        try:
-            server = self._server_socket(unix_socket=path, keyfile=SERVER_KEY, certfile=SERVER_CERT)
-            self._assert_connection_success(server, path=path, cert_reqs=ssl.CERT_NONE)
-        finally:
-            os.unlink(path)
```

Comment:
We should still be removing it after the test (finally).  It makes sense to remove it before as you have added.

Reply:
you're right.